spot_wrapper/wrapper.py

```diff
@@ -1574,6 +1604,21 @@ def navigate_to(
 
         return resp
 
+    @try_claim
+    def navigate_to_dynamic(self, navigate_to):
+        """Navigate with graph nav. Use this instead of navigate_to if you want to modify Spot's speed during navigation commands
+         and pause/cancel navigations (using handle_set_navigate_to_params in spot_ros2). Unlike navigate_to, this methods assumes
```

Comment:
This should preferably explain the mechanism by which the dynamic speed is controlled by referencing stuff in the wrapper rather than in the ROS code.

Also check spelling in this section.

Reply:
Fixed! I've moved set_navigate_to_params to the wrapper and updated the docstrings accordingly.